frontend/src/app/features/map/FindMeControl.tsx

```diff
@@ -0,0 +1,35 @@
+import { useDispatch } from 'react-redux';
+import { IconButton } from '@mui/material';
+import clsx from 'clsx';
+
+import { setMyLocationVisible, useMyLocationVisible } from './map-slice';
+import { useGeolocationPermission } from '../../../hooks/useMyLocation';
+
+import { FindMe } from '../../components/common/icon';
+
+export function FindMeControl() {
```

Comment:
`FindMeControl` and `FindMeButton` are almost identical. To avoid code duplication, they should be combined into a single component that accepts settings for its appearance/position as props.

Reply:
The two are only similar to check for position but are very much different in terms of returning the component altogether. Its just not only the difference in styling which could be passed as props. I can definitely create a common component but it will a lot of overhead to pass numerous props and the added advantage is only these 2 components. Not sure if that would be a very effective approach. We can discuss more offline.